keras_eval/eval.py

```diff
@@ -337,14 +339,39 @@ def _predict_folder(self, folder_path, verbose):
             # Predict
             if verbose:
                 print('Making predictions from model ', str(i))
-            probabilities.append(model.predict(images, batch_size=self.batch_size, verbose=1))
+            probabilities.append(model.predict(images, batch_size=self.batch_size, verbose=verbose))
 
         self.probabilities = np.array(probabilities)
         self.combined_probabilities = utils.combine_probabilities(self.probabilities, self.combination_mode)
         self.image_paths = image_paths
 
         return self.probabilities
 
+    def _predict_list(self, image_list, verbose=True):
+        '''
+
+        Predict the class probabilities of a set of images from a list.
+
+        Args:
+            image_list: List of images
+
+        Returns: Probabilities predicted, image path for every image (aligned with probability)
+
+        '''
+        probabilities = []
+        image_paths = []
+
+        for image_path in image_list:
```

Comment:
Should we first check if the `image_list` is correct? i.e. checking that files exists